src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -86,6 +88,7 @@ const ConfirmTransactionModal = ({
   onClose,
 }: OwnProps): React.ReactElement | null => {
   const dispatch = useDispatch()
+  const ethToken = useToken(ETH_ADDRESS) as Token | null
```

Comment:
it feels weird to me (coming from more strongly typed languages) that we cast the return type here ... we are ignoring the NFT type and I am also not sure if `undefined` and `null` are the same :/